mongodbatlas/resource_mongodbatlas_privatelink_endpoint_service.go

```diff
@@ -196,8 +204,8 @@ func resourceMongoDBAtlasPrivateEndpointServiceLinkDelete(d *schema.ResourceData
 		}
 
 		stateConf := &resource.StateChangeConf{
-			Pending:    []string{"NONE", "PENDING_ACCEPTANCE", "PENDING", "DELETING", "WAITING_FOR_USER"},
-			Target:     []string{"REJECTED", "DELETED"},
+			Pending:    []string{"NONE", "PENDING_ACCEPTANCE", "PENDING", "DELETING", "WAITING_FOR_USER", "INITIATING"},
+			Target:     []string{"REJECTED", "DELETED", "FAILED"},
```

Comment:
Same as above: Related to the docs comment I made and fix you did in the last commit, do we have states here that are no longer applicable?